src/simulate/main.py

```diff
@@ -23,9 +25,12 @@ def main():
     args = argparse.ArgumentParser()
     args.add_argument('property_files', help='path to the config.yml file')
     args.add_argument('directory_json_files', help='path to directory that contains json files')
+    args.add_argument('times_to_run', nargs='?', help='number representing how many times to send the static events',
+                      type=int, default=1)
```

Comment:
Some (all?) of the changes in this PR also appear in the previous one.  Only changes that pertain to this effort (containerizing) should be in this PR.

Reply:
While working on containerizing the app a thought occurred that this argument can have a default value and should be made optional So I have this in place.

Reply:
Also, this PR  is a subbranch of TLCALIPER-126 ( which is not merged and in code review) and showing the difference against the Master branch so you may find changes from TLCALIPER-126 also here. Line 33 below is an example of that

Reply:
I think using subbranches is making problems for you.